validator/cpp/htmlparser/WORKSPACE

```diff
@@ -10,9 +10,9 @@ http_archive(
 
 http_archive(
     name = "com_google_googletest",
-    sha256 = "f44d8f9456e01fcaacc1930d37a3d8c74c295d603658db9abd797235bb648266",
-    strip_prefix = "googletest-486a4a6be5565e89ae0d72f532c441423110bc9e",
-    urls = ["https://github.com/google/googletest/archive/486a4a6be5565e89ae0d72f532c441423110bc9e.zip"],  # 2019-01-07
+    sha256 = "2e51918ba711aa489a2f9b5778108875437bac5960f37b5d56e1ca0d09d725eb",
+    strip_prefix = "googletest-4400d68de42a77d9709221cb8f179f40c0a21732",
+    urls = ["https://github.com/google/googletest/archive/4400d68de42a77d9709221cb8f179f40c0a21732.zip"],  # 2019-01-07
```

Comment:
@ampproject/wg-caching, given that the commit hash is being auto-updated by renovate, does it make sense to remove the trailing comment? I'd imagine the hash is far newer than the one from 2019-01-07.